### ISSUE TITLE
[Elements/buttons] The border focus color and size on buttons could be problematic

### ISSUE THREAD

**Issue by AccessForAll** [18F/govt-wide-patternlibrary#166]
* Reconsider specifying the focus color around buttons.
     * We don’t know if the yellow color will provide sufficient contrast on other navigable items, and the color probably should stay the same around all navigable items.
     * This is particularly important for people who navigate by keyboards, non-mouse devices, or assistive technologies.

* The border should be thicker to help people who have aging eyes or low-vision.

**Comment by maya** [comment]
The border width of focus state is currently 2px and agree this could be bigger.

**Comment by carodew** [comment]
I believe the colors shown in the library are 'for now' while @angel is working on the branding and color scheme. But I agree that the yellow is problematic.

**Comment by colinpmacarthur** [comment]
Many thanks! I have added this issue to our project's "icebox" so it may be addressed in future versions of the library.
